<launch>
    <node name="bno" pkg="wurov2_0" type="bno_node.py"/>
    <node name="thrusters" pkg="wurov2_0" type="thruster_node.py"/>
    <node name="cam" pkg="wurov2_0" type="camera_node.py"/>


 <group ns="camera3">
  <node name="usb_cam3" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
    <param name="framerate" value="40"/>
  </node>
  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam/image_raw"/>
    <param name="autosize" value="true" />
  </node>
 </group>
</launch>